.github/workflows/release-it.yml

```diff
@@ -0,0 +1,36 @@
+# #########################################################################
+# Creates a new release using `workflow_dispatch` event trigger with `type`
+# as input to describe the type of release to create
+name: 'Release-it: Create a new release on demand'
+
+on:
+  workflow_dispatch:
+    inputs:
+      type:
+        description: 'Type. Can be `patch` `minor` or `major`'
+        required: true
+        default: 'patch'
+jobs:
+  release-it:
```

Comment:
You should limit this action to whoever you're comfortable with to do releases.

> To trigger a workflow in a repository, the user should be a collaborator with Write permission in the repository. Normally the external users can’t trigger workflows in the repository.

Reply:
Fixed! Thanks :)